gulp-core-build/src/jsonUtilities/SchemaValidator.ts

```diff
@@ -66,6 +58,14 @@ export class SchemaValidator {
     return rawConfig as TResult;
   }
 
+  private static getFormattedErrorMessage(errors: Validator.SchemaErrorDetail[], dataFilePath?: string): string {
```

Comment:
Nope. I can't do this because then the Validator.SchemaErrorDetail  will still be throwing errors